contrib/cf-remote/README.md

```diff
@@ -85,6 +85,17 @@ Note that this demo setup (`--demo`) is notoriously insecure.
 It has default passwords and open access controls.
 Don't use it in a production environment.
 
+Note also that for distributions that we don't have packages for you might add entries to
+`/etc/os-release` on the target system to give cf-remote an alternative. For example, PureOS
+is based on debian testing, so currently the appropriate package we create would be debian 10.
+
+```
+ID_LIKE="debian"
+VERSION_ID_LIKE="10.0"
```

Comment:
I dunno about adding this custom one to os-release. Feels fiddly, and something that may not be retained through OS upgrades, so they would also need to deploy policy to ensure the custom entries are always re-added.

More complex, but a better user experience IMHO wold be to prompt the user to pick the most similar platform then cache that info to re-use for any future same case.

Reply:
@nickanderson I don't think cf-remote should turn interactive by default. We could add a better error message, better documentation, or `--interactive` option, but IMO, this is pretty low priority.

@craigcomstock FYI, I would just do:

```
olehermanse@OH-WIN ~ $ cf-remote packages debian 64 client
Available releases: 3.15.x, 3.15.2, 3.15.1, 3.15.0, 3.15.0b1, 3.12.x, 3.12.5, 3.12.4, 3.12.3, 3.12.2, 3.12.1, 3.12.0
Using 3.15.2 LTS (default):
Package already downloaded: '/home/olehermanse/.cfengine/cf-remote/packages/cfengine-nova_3.15.2-1.debian7_amd64.deb'
Package already downloaded: '/home/olehermanse/.cfengine/cf-remote/packages/cfengine-nova_3.15.2-1.debian8_amd64.deb'
Package already downloaded: '/home/olehermanse/.cfengine/cf-remote/packages/cfengine-nova_3.15.2-1.debian9_amd64.deb'
Package already downloaded: '/home/olehermanse/.cfengine/cf-remote/packages/cfengine-nova_3.15.2-1.debian10_amd64.deb'
olehermanse@OH-WIN ~ $ cf-remote install --client-package /home/olehermanse/.cfengine/cf-remote/packages/cfengine-nova_3.15.2-1.debian9_amd64.deb ...
```